markdown/serializers.py

```diff
@@ -162,8 +162,13 @@ def _serialize_html(write, elem, format):
                     write(_escape_cdata(text))
             for e in elem:
                 _serialize_html(write, e, format)
-            if tag.lower() not in HTML_EMPTY:
-                write("</" + tag + ">")
+            write("</" + tag + ">")
+        elif tag.lower() in HTML_EMPTY:
+            if format == "xhtml":
+                write(" /")
```

Comment:
If you look at the coverage report, this line never gets run. In fact, this case is caught in line 154. Perhaps the `html` case could be moved up there as well and we can remove this repeat check of `HTML_EMPTY`

Reply:
I have removed this line, I ran coverage as you said and this was no longer needed.